python/grass/docs/src/gunittest_running_tests.rst

```diff
@@ -78,11 +78,31 @@ use::
 
     python -m grass.gunittest.main ... --min-success 60
 
-If all tests should succeed, use ``--min-success 100``. If you want
+If all tests should succeed, use ``--min-success 100`` (the default). If you want
 to run the test and ``grass.gunittest.main`` returning zero return code
 even if some tests fail, use ``--min-success 0``
 
 
+Excluding test files from testing
+---------------------------------
+
+Test files which would be otherwise collected and executed can be excluded
+from the testing using the ``exclude`` key in the ``.gunittest.cfg``
+configuration file under the ``.gunittest`` section.
+The value of ``exclude`` is whitespace-separated list of exclude files
+possibly with wildcards (from Python fnmatch).
+Directory separators are converted to ``/`` for the matching.
+Paths are relative to the directory the testing was started from.
+
+For example, to exclude the whole directory ``vector`` and one specific file,
+you would use::
```

Comment:
```suggestion
For example, to exclude the whole directory ``vector`` and one specific
raster file, you would use::
```

Reply:
Hm, raster file sounds like a GeoTIFF or something, leaving as is.